CHANGELOG.md

```diff
@@ -11,6 +11,9 @@
 
 * **[Fix]** Fix crash involving SDK's `ms_viewWillAppear` method.
 
+### App Center Data
```

Comment:
Change 2.3.1 above to 2.4.0

Reply:
Yep, breaking change on a preview module @MehrdadMzfr .